datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -23,6 +24,11 @@ def ensure_unicode(s):
 to_string = ensure_unicode if PY3 else ensure_bytes
 
 
+def round_value(value, sig_digits="0.01", rounding_method=ROUND_HALF_UP):
+    getcontext().rounding = rounding_method
+    return float(Decimal(str(value)).quantize(Decimal(sig_digits)))
```

Comment:
```
def round_num(value, precision=0):
    return float(Decimal(str(value)).quantize(Decimal('0.{}'.format('0' * precision)), rounding=ROUND_HALF_UP))
```

Reply:
I like leaving it open to modify the rounding strategy if anyone requires something like Python 3s ROUND_HALF_EVEN function later. I don't think it hurts to leave it in the signature as an optional parameter

I do like the `precision` component!